packages/web/src/common/store/pages/feed/lineup/sagas.ts

```diff
@@ -29,7 +33,7 @@ const getAccountUser = accountSelectors.getAccountUser
 
 type FeedItem = LineupTrack | Collection
 
-const filterMap = {
+const filterMap: { [k in FeedFilter]: full.GetUserFeedFilterEnum } = {
```

Comment:
Could you achieve the same with `as const`?

Reply:
Oh potentially. I never use `as const` but the goal was to make sure this breaks if we don't have an item for each key in the enum